contrib/drivers/drivers.go

```diff
@@ -129,3 +129,30 @@ func Clean(d VolumeDriver) VolumeDriver {
 
 	return d
 }
+
+
+
+type MetricDriver interface {
```

Comment:
Please use `go fmt` to update the code style.

Reply:
@leonwanghui Fixed!. Please check